tests/File/CommonFileTest.php

```diff
@@ -279,13 +279,16 @@ public function testFilesPermissions(): void
         $this->assertEmpty($newTokenClient->listFiles());
 
         $newFileId = $this->createAndWaitForFile($filePath, $uploadOptions, $newTokenClient);
+        /** @var array<array{id:int}> $files */
         $files = $newTokenClient->listFiles();
         $this->assertCount(1, $files);
-        $this->assertEquals($newFileId, reset($files)['id']);
+        /** @var array{id:int} $reset */
+        $reset = reset($files);
+        $this->assertEquals($newFileId, $reset['id']);
 
         // new file should be visible for master token
         $files = $this->_client->listFiles();
-        $this->assertEquals($newFileId, reset($files)['id']);
+        $this->assertEquals($newFileId, $reset['id']);
```

Comment:
@zajca pride mi, ze tu chyba este znovu:

```php
$reset = reset($files);
```
lebo to robi novy fetch

Reply:
Ty jo dobrý catch, máš pravdu, fixnu to.

Reply:
Mozeme to prihodim k tomu mojmu PR? https://github.com/keboola/storage-api-php-client/pull/910